### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by danopdev**
Just a small public method documentation.
The method does not exists on 3.x so I propose it for 4.x only.

### Pull Request Readiness Checklist

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [ x ] I agree to contribute to the project under Apache 2 License.
- [ x ] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [ x ] The PR is proposed to the proper branch
- [ ] There is a reference to the original bug report and related work
- [ ] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [ ] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

**Comment by MuhammedKAldin** [comment]
why was the merging blocked when it's an open source repo?

**Review by alalek — COMMENTED** [review]
Thank you for contribution!

File: modules/stitching/include/opencv2/stitching.hpp [hunk 1]
```diff
@@ -302,6 +302,12 @@ class CV_EXPORTS_W Stitcher
     std::vector<int> component() const { return indices_; }
     std::vector<detail::CameraParams> cameras() const { return cameras_; }
     CV_WRAP double workScale() const { return work_scale_; }
+
+    /** @brief Return the mask of the panorama as 8U UMat: 0xFF (white) for pixels filled by the input images,
+    0 (black) for unused pixels. It can be used as the mask for inpaint.
```
[the remark this answers is not in the document]
Comment (danopdev) [reply modules/stitching/include/opencv2/stitching.hpp]: It's done.
Please let me know if I'm missing something.

**Review by alalek — APPROVED** [review]
Thank you!

**Pull request merged**